ecl/hthor/hthor.cpp

```diff
@@ -1211,6 +1229,7 @@ void CHThorIndexWriteActivity::execute()
         builder->finish(metadata, &fileCrc);
         duplicateKeyCount = builder->getDuplicateCount();
         cummulativeDuplicateKeyCount += duplicateKeyCount;
+        numDiskWrites = io->getStatistic(StNumDiskWrites);
```

Comment:
Slight question about if this was executed in a child query, but that is so unusual that it doesn't matter...